ethereum/core/src/test/java/org/hyperledger/besu/ethereum/mainnet/DefaultTimestampScheduleTest.java

```diff
@@ -86,17 +86,6 @@ public void getByBlockHeader_whenSpecNotFoundReturnsNull() {
     assertThat(schedule.getByBlockHeader(BLOCK_HEADER)).isNull();
   }
 
-  @Test
-  public void streamMilestoneBlocksReturnTimestampsInOrder() {
```

Comment:
Think this test was important, has it been replace by another one? I think isOnMilestoneBoundary has a different purpose and and would be nice to have a test to always guarantee we are still delivering the forks and the right order.

Reply:
streamMilestoneBlocks isn't used in production any more, so do we want tests for all of our test-support code is the really the question here...

Reply:
I agree, we should not test a method that is not being used anymore, but how do we guarantee the forks are in the right order now? Is there a separate test for that somewhere  else?

Reply:
Yes, the order is tested in the ScheduleBuilderTests: https://github.com/hyperledger/besu/blob/main/ethereum/core/src/test/java/org/hyperledger/besu/ethereum/mainnet/TimestampScheduleBuilderTest.java#L61
https://github.com/hyperledger/besu/blob/main/ethereum/core/src/test/java/org/hyperledger/besu/ethereum/mainnet/TimestampScheduleBuilderTest.java#L104

There is also some validation in the code itself: https://github.com/hyperledger/besu/blob/main/ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/TimestampScheduleBuilder.java#L56

The above test was covering that streamMilestoneBlocks returned them in the same order.